src/definitions/modules/dropdown.js

```diff
@@ -3294,6 +3294,9 @@ $.fn.dropdown = function(parameters) {
           menu: function() {
             return ($menu.length > 0);
           },
+          subMenu: function() {
+            return $menu.find(selector.menu).length > 0;
```

Comment:
This has to respect a multilevel submenu
```suggestion
          subMenu: function($subMenu) {
            return ($subMenu || $menu).find(selector.menu).length > 0;
```